pipelines/openjdk10_nightly_pipeline.groovy

```diff
@@ -0,0 +1,42 @@
+println "building OpenJDK10"
+stage 'build OpenJDK'
```

Comment:
I would recommend using braces around a stage to clearly identify what the stage encapsulates
```
stage ('build OpenJDK') {
    parallel Platforms
}
```

Reply:
Fixed (will push shortly)